packages/lmi/tests/test_llms.py

```diff
@@ -391,6 +391,90 @@ def test_pickling(self, tmp_path: pathlib.Path) -> None:
         assert llm.config == rehydrated_llm.config
         assert llm.router.deployment_names == rehydrated_llm.router.deployment_names
 
+    @pytest.mark.asyncio
+    async def test_acompletion_iter_logprobs_edge_cases(self) -> None:
+        """Test that acompletion_iter handles various logprobs edge cases gracefully."""
+        model = LiteLLMModel(name=CommonLLMNames.OPENAI_TEST.value)
+        messages = [Message(content="Say hello", role="user")]
```

Comment:
```suggestion
        messages = [Message(content="Say hello")]
```

`"user"` is the default